tests/test_api.py

```diff
@@ -0,0 +1,7 @@
+import sds_api
+import os
+
+current_directory = os.path.dirname(os.path.realpath(__file__))
+
+def test_upload():
```

Comment:
Is it possible to write some tests? Or is there not really a consistent SDS backend for testing? It might be worth at least testing if the correct queries / URLs / login values are generated, even if they aren't then actually sent to AWS.